### ISSUE TITLE
script file and data file needs to be provided in certain order

### ISSUE THREAD

**Issue by alperyilmaz** [ezrosent/frawk#65]
When I was running the file `info/scripts/groupby.sh` I encountered an error at the following line:
```bash
time $FRAWK -bllvm -F'\t' -f "$SCRIPT_FILE" "${TSV}"
```
When I was trying to trıubleshoot the problem I noticed that
```bash
cat datafile | frawk -f scriptfile   # works
frawk datafile -f scriptfile         # works
frawk -f scriptfile datafile         # fails
```
after trying different options, I found out that in case of  `frawk -f scriptfile datafile` the datafile needs to be provided with `--`. Let me demonstrate the case:

frawk works fine with pipe:
```bash
$ echo -e "a,b,c\n1,2,3\n4,5,6"
a,b,c
1,2,3
4,5,6

$ echo -e "a,b,c\n1,2,3\n4,5,6" | frawk -icsv '{ print $2; }'
b
2
5
```
If we save the script as file, it woks fine in pipe
```bash
$ echo '{ print $2; }' > test.awk
$ echo -e "a,b,c\n1,2,3\n4,5,6" | frawk -icsv -f test.awk
b
2
5
```
if both data and script are from file, then fails
```bash
$ echo -e "a,b,c\n1,2,3\n4,5,6" > test.dat
$ frawk -icsv -f test.awk test.dat
Unrecognized token `,` found at line 3, column 4:line 3, column 5
Expected one of "\n", "[" or "{"
```
workaround is using `--`
```bash
$ frawk -icsv -f test.awk -- test.dat
b
2
5
```
Not sure if this is bug or a feature but still it would be great if `frawk` can be drop-in replacement for awk, by simply replacing awk with frawk in scripts , everything should work. So, it would be great if frawk can run just like awk where datafile is provided after script file.
  
I'm using frawk version 0.4.2 in Linux.

**Comment by ezrosent** [comment]
Hi! I think this is fixable. I agree that frawk should be close to other awks, where possible. I'll follow up soon.

**Comment by ezrosent** [comment]
I believe this is fixed in the commit above, which is included in frawk version 0.4.4. Let me know if you still notice problems along these lines, and thanks again for the report.

**Comment by alperyilmaz** [comment]
compiled 0.4.4 and issue is fixed.. thanks..
